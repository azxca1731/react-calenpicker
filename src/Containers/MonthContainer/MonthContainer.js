import React from "react";
import PropTypes from "prop-types";

import { CssConnector } from "Containers/Provider";
import Month from "Components/Month";

const MonthContainer = props => (
  <Month cssObject={props.cssObject} month={props.month} />
);

MonthContainer.propTypes = {
  month: PropTypes.string.isRequired,
  cssObject: PropTypes.object
};

export default CssConnector(({ state }) => ({
  cssObject: state.MonthCssObject
}))(MonthContainer);
