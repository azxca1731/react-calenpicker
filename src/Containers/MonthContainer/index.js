import MonthContainer from "./MonthContainer";

export default MonthContainer;
