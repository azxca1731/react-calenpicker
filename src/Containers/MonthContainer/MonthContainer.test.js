import React from "react";
import { shallow } from "enzyme";
import MonthContainer from "./index";

describe("MonthContainer", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<MonthContainer />);
    expect(wrapper).toHaveLength(1);
  });
});
