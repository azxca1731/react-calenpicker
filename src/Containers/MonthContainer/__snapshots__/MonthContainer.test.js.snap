// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthContainer renders with snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
